### PR TITLE
Add stories for LocationMove & adapt TaskListItem story

### PR DESCRIPTION
**Pull request by shaun-chiang**
This is a storybook entry for upcoming PR for TaskListItem - splitting it out to LocationMove.
Screen Shot 2019-10-18 at 12 10 56 PM
Screen Shot 2019-10-18 at 12 11 07 PM
After that is merged, revert package.json changes.

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -121,6 +128,37 @@ storiesOf('TaskListItem', module)
     </CenterView>
   ));
 
+storiesOf('LocationMove', module)
```
Comment (saurabhgangarde) [inline stories/index.js]: I think we also support with just origin location and destination location missing, might wanna add one case with just 1 location.

Comment (shaun-chiang) [reply stories/index.js]: yup, below there is story "without destination location" which just shows the origin location.
Not sure if we should include with both origin and destination missing as that would show nothing 😅

Comment (saurabhgangarde) [reply stories/index.js]: Ohh yes.. I kept reading with, with, with ..

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "react": "16.3.1",
     "react-native": "0.55.4",
-    "rn-wh-components": "git+https://github.com/Redmart/rn-wh-components.git"
+    "rn-wh-components": "git+https://github.com/Redmart/rn-wh-components.git#feature/modular-tasklistitem"
```
Comment (saurabhgangarde) [inline package.json]: May be we should merge the rn components PR first and then get rid of this. Cauase eventually "feature/modular-tasklistitem" branch will be deleted after PR is merged ..

Comment (shaun-chiang) [reply package.json]: yup, this is the approach described in PR description:

> After that is merged, revert package.json changes.

**Pull request merged**